Simplified/NYPLAnnouncementManager.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+private let announcementKey: String = "NYPLPresentedAnnouncementKey"
+private let filename: String = "NYPLPresentedAnnouncementsList"
+
+class NYPLAnnouncementManager {
+  private static var presentedAnnouncement: Set<String> = Set<String>()
+    
+  class func presentAnnouncements(_ announcements: [Announcement]) {
+    for announcement in announcements {
+      if shouldPresentAnnouncement(id: announcement.id) {
+        DispatchQueue.main.async {
+          let vc = NYPLAnnouncementViewController(announcement: announcement)
+          NYPLRootTabBarController.shared()?.safelyPresentViewController(vc, animated: true, completion: nil)
+        }
+      }
+    }
+  }
+  
+  // MARK: - Read
+    
+  class func shouldPresentAnnouncement(id: String) -> Bool {
```

Comment:
can this be fileprivate? You could add a `testing_shouldPresent...` wrapper in an extension if you need it for testing.